components/item-info.js

```diff
@@ -140,6 +142,8 @@ export default function ItemInfo ({
           !item.mine && !item.deletedAt && <DontLikeThisDropdownItem id={item.id} />}
         {item.mine && !item.position && !item.deletedAt &&
           <DeleteDropdownItem itemId={item.id} type={item.title ? 'post' : 'comment'} />}
+        {(item.parentId || item.text) && onQuoteReply &&
```

Comment:
If it has a parent, or if it has text, and if we have a handler for the quote reply action, render the menu item.